src/libtools/my_x11_conv.c

```diff
@@ -1495,20 +1495,20 @@ void* inplace_XRRMonitorInfo_enlarge(void* a, int n)
     if(a) {
         my_XRRMonitorInfo_32_t* src = a;
         my_XRRMonitorInfo_t* dst = a;
-        for(int i=0; i<n; ++i) {
-            dst->name = from_ulong(src->name);
-            dst->primary = src->primary;
-            dst->automatic = src->automatic;
-            dst->noutput = src->noutput;
-            dst->x = src->x;
-            dst->y = src->y;
-            dst->width = src->width;
-            dst->height = src->height;
-            dst->mwidth = src->mwidth;
-            dst->mheight = src->mheight;
-            dst->outputs = from_ptrv(src->outputs);
+        for (int i = 0; i < n; ++i) {
```

Comment:
Mmmm, actually, that for loop should be inverted too, else, if there is more then 1 element, the enlarge will overwrite itself.

Reply:
I'll fix it in a another commit

Reply:
Indeed! Thanks. In the cs1.6 case, n is 1.